lib/src/utils/blockfrost_signature_validator.dart

```diff
@@ -0,0 +1,115 @@
+import 'dart:convert';
+
+import 'package:blockfrost_api/src/utils/signature_validator.dart';
+import 'package:crypto/crypto.dart';
+
+const int maxToleranceSeconds = 60;
+
+/// Adapter class which implements the validator interface to validate the blockfrost webhook signature.
+class BlockfrostSignatureValidator implements SignatureValidator {
+  @override
+  bool validate({
```

Comment:
Can you please change the order of the params to match the node.js, python and ruby SDK implementation

https://github.com/blockfrost/blockfrost-python/blob/master/blockfrost/helpers.py#L18
https://github.com/blockfrost/blockfrost-js/blob/master/src/utils/helpers.ts#L211